share/functions/choose.fish

```diff
@@ -0,0 +1,6 @@
+function choose --description "Chooses a random item from a list"
```

Comment:
I'd prefer to see this implemented via `random choose` (or `random choice` or `random select`) via a `random` function. See what we do with the `history` function to augment the history builtin. Adding new commands runs the risk of causing problems for someone with an external command by the same name. So we should not do so if there is a reasonable alternative.